elements/core/components/Button/Button.styles.ts

```diff
@@ -45,6 +46,14 @@ export const Button = styled.button<{
     }
   `}
 
+  ${(props) => props.iconRight && css`
```

Comment:
there you can give the button:
display: flex;
flex-direction: row;

and when property iconRight, then set flex-directiopn to row-reverse;